platform/shared/src/lib.rs

```diff
@@ -158,6 +162,39 @@ impl LoggerHolder {
     holder.shutdown(false);
     drop(holder);
   }
+
+  /// Logs an out-of-the-box app launch TTI log event. The method should be called only once.
+  /// Consecutive calls have not effect.
+  pub fn log_app_launch_tti(&self, duration: time::Duration) {
+    self.app_launch_tti_log.call_once(|| {
```

Comment:
I'm just learning about this primitive and their docs say 
>This method will block the calling thread if another initialization routine is currently running.
which seems to be aimed to help with
>It is also guaranteed that any memory writes performed by the executed closure can be reliably observed by other threads at this point

This sounds like an overkill to me since there are no shared resources that we'd like to guard, but maybe I'm misunderstanding something

Reply:
sg